lua/lsp/init.lua

```diff
@@ -0,0 +1,16 @@
+require("lsp.handlers").setup()
+
+-- Setup language servers.
+function check_start_kcl_lsp()
+  local kcl_lsp_config = {
+    name = 'kcl-cli',
+    cmd = {'kcl-language-server'},
+    filetypes = { 'k' }
```

Comment:
I would name the filetype `kcl`

generally in vim we use bigger names for filetype.. EG `python`, `rust`, `yaml` etc

Then we can associate  `*.k` with `filetype=kcl`

eg

```lua
--- map .k file to filetype "kcl"
vim.filetype.add({
    extension = {
        k = "kcl",
    },
})
```

Reply:
Thanks for your suggestion. I have added this part in `lua/basic.lua`.